Contributers/SumitSinghRaghuwanshi1245.md

```diff
@@ -0,0 +1,87 @@
+## <h1> **Sumit Singh Raghuwanshi** </h1>
+Web Developer | MERN Stack | WEB3 Enthusiast
+
+## 🌐 Links
+-  Email:   [ssraghuwanshi1245@gmail.com](ssraghuwanshi1245@gmail.com)  
+-  LinkedIn:   [Sumit Singh Raghuwanshi](https://www.linkedin.com/in/sumit-singh-raghuwanshi/)  
+-  GitHub:   [SumitSinghRaghuwanshi1245](https://github.com/SumitSinghRaghuwanshi1245)  
+-  X:   [Sumit_Singh_R](https://x.com/Sumit_Singh_R)  
+
+---
+
+## 📌 Professional Summary
+I am an **undergrad student pursuing B.Tech in Computer Science and Business System**. I'm a **frontend developer**, and also have a keen interest in **AI, Web3, and Blockchain**. Passionate about exploring emerging technologies and building innovative solutions.
+
+---
+
+## 🎓 Education
+- **B.Tech in Computer Science and Business System**  
+  *SoIT, Rajiv Gandhi Prodyogiki Vishwavidyalaya (2023-2027)*  
+
+
+- **Senior Secondary Education**  
+  *St. Paul Sr. Sec. Co-Ed School, Bhopal*  
+
+---
+
+## 💻 Technical Skills
+### 🔹Languages :  *HTML, CSS, JavaScript, C, Java*  
+
+
+### 🔹 Frontend Technologies  : *React.js, Tailwind CSS, Redux Toolkit* 
+
+
+### 🔹 Backend Technologies  : *Node.js, Express.js*
+ 
+
+### 🔹 Databases  : *MySQL, MongoDB*
+  
+
+### 🔹 Version Control  : *Git, GitHub*
+  
+
+### 🔹 IDE  : *Visual Studio Code*
+ 
+
+### 🔹 Others  : *Content Writing, Prompt Engineering, Research*  
+
+
+---
+
+## 💼 Experience
+
+### 🔹 Open Source Contribution  
+### **Social Winter of Code**  
+(Jan 2025)  
+*Contributed to the SWoC project.* [(link)](https://github.com/SumitSinghRaghuwanshi1245/Job-Hive)
+
+
+
+### 🔹 **Content Writer (Volunteer)** 
+### **IEEE SoIT RGPV**
+
+ (May 2024 - Oct 2024)  
+*Wrote technical as well as non-technical content, blogs, newspapers, and content for posts, etc., for IEEE SOIT RGPV.*  
+
+ 
+
+
+
+---
+
+## 🏗️ Projects
```

Comment:
Expand the projects description, add what you learnt, what tech you used, what problem does it solve